pages/c/_cluster/apps/charts.vue

```diff
@@ -292,22 +307,31 @@ export default {
     </header>
 
     <div class="left-right-split">
-      <div class="mt-10">
-        <Checkbox
-          :value="allRepos"
-          :label="t('catalog.charts.all')"
-          :class="{'pull-left': true, 'repo': true}"
-          @input="toggleAll($event)"
-        />
-        <Checkbox
-          v-for="r in repoOptions"
-          :key="r.label"
-          v-model="r.enabled"
-          :label="r.label"
-          :class="{'pull-left': true, 'repo': true, [r.color]: true}"
-          @input="toggleRepo(r, $event)"
-        />
+      <div>
+        <Select
+          :searchable="false"
+          :options="[{label: 'All', all: true, enabled:areAllEnabled() },...repoOptions]"
+          :value="flattenedRepoNames"
+          class="option-label-container "
+          @option:selecting="$event.all ? toggleAll(!$event.enabled) : toggleRepo($event, !$event.enabled) "
+        >
+          <template #selected-option-container="{option}">
+            <span class="option-label">
+              {{ option.label }}
+            </span>
+          </template>
+
+          <template #option="repo">
+            <Checkbox
+              :value="repo.enabled"
+              :label="repo.label"
+              :class="{'pull-left': true, 'repo': true, [repo.color]: true}"
```

Comment:
We've moved away from the gradient style colouring that shows up in the background of each option. It's good that there's some colour here to tie in with the top border on the cards, but needs another approach.

Reply:
Something like this @lvuch ? 
![Screen Shot 2021-05-21 at 11 15 17 AM](https://user-images.githubusercontent.com/42977925/119180969-daa90b80-ba25-11eb-93a5-263ebe067867.png)

Reply:
@mantis-toboggan-md yep looks good